src/+otp/+kuramotosivashinsky/KuramotoSivashinskyProblem.m

```diff
@@ -0,0 +1,59 @@
+classdef KuramotoSivashinskyProblem < otp.Problem
+    
+    methods
+        function obj = KuramotoSivashinskyProblem(timeSpan, y0, parameters)
+            obj@otp.Problem('Kuramoto-Sivashinsky', [], timeSpan, y0, parameters);
+        end
+    end
+    
+    methods
+        function soly = solution2real(soly)
```

Comment:
To be consistent with Pendulum problem, let's rename to something like `convert2grid`